ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -129,10 +130,20 @@ public KsqlRestApplication(
     this.statusResource = statusResource;
     this.streamedQueryResource = streamedQueryResource;
     this.ksqlResource = ksqlResource;
-    this.isUiEnabled = isUiEnabled;
+
     this.versionChckerAgent = versionCheckerAgent;
 
     this.commandRunnerThread = new Thread(commandRunner);
+    final String ksqlInstallDir = System.getenv("KSQL_INSTALL_DIR");
+    if (ksqlInstallDir == null && isUiEnabled) {
+      log.info("environment variable KSQL_INSTALL_DIR is not defined. "
```

Comment:
Shouldn't this be an error or at least `WARN`?

Also not sure of the hygiene of directly referencing and using environment variables like this. Might be better to pass it as a system property specified on the CLI? That seems like a more direct way to achieve the same purpose.

Reply:
Yep - i was 50/50 on the env var versus system property. I'm happy to change it. 
I'll change the log level